src/styles/base/null.scss

```diff
@@ -0,0 +1,62 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
```

Comment:
```suggestion
* {
  box-sizing: border-box;
}
```

Reply:
it is just a base null-style file. ok, whatever